tensorflow/core/kernels/random_op_cpu.h

```diff
@@ -86,7 +86,13 @@ struct FillPhiloxRandomTask<Distribution, false> {
                   int64 start_group, int64 limit_group, Distribution dist) {
     const int kGroupSize = Distribution::kResultElementCount;
 
-    gen.Skip(start_group);
+    // Decide skip strides according to different kResultElementCount:
+    // * `1 = (4 + 3) / 4` for normal Distribution.
+    // * `1 = (2 + 3) / 4` for double/int64 Distribution.
+    // * `4 = (16 + 3) / 4` for vecotorized float/bfloat16 Distribution.
```

Comment:
Nit: typo.
```suggestion
    // * `4 = (16 + 3) / 4` for vectorized float/bfloat16 Distribution.
```

Reply:
Sure, I'll avoid force-pushing next time and use merge to update branch, thanks for the reminder!

Reply:
That would be great. Thank you so much!